src/bitcoin-chainstate.cpp

```diff
@@ -0,0 +1,255 @@
+#include <iostream> // for cout and shit
```

Comment:
In commit "build: Add example bitcoin-chainstate executable" (01840e6161396ed0ab30a122921f67b8dce45499)

Should add the commit description "The bitcoin-chainstate executable serves to..." to the top of this file so it is not lost in git history. Or `bitcoin-chainstate` could accept "-help" option and describe its purpose there.

Reply:
Done as of 0d90128a3b6acab0fd9d1bc39518e2783b55fdf3